runtime/js-compute-runtime/js-compute-builtins.cpp

```diff
@@ -1129,8 +1129,8 @@ bool process_pending_request(JSContext *cx, HandleObject request) {
     return false;
   }
 
-  RootedObject response(
-      cx, builtins::Response::create(cx, response_instance, response_handle, body, true));
+  RootedObject response(cx, builtins::Response::create(cx, response_instance, response_handle, body,
+                                                       true, false, nullptr));
```

Comment:
One pattern that I like for boolean arguments like this is to create a local with a more informative name, and then use that as the argument.
```suggestion
  bool is_upstream = true;
  bool is_grip_upgrade = false;
  RootedObject response(cx, builtins::Response::create(cx, response_instance, response_handle, body,
                                                       is_upstream, is_grip_upgrade, nullptr));
```

Reply:
This is the correct pattern for sure 👍 
I then started to wonder if we could/should use the named parameters pattern -- https://pdimov.github.io/blog/2020/09/07/named-parameters-in-c20/